web/src/org/openmrs/module/chica/web/ManualCheckinSSNMRN.java

```diff
@@ -38,8 +31,6 @@
 	private static final String XML_FALSE = "false";
 	private static final String XML_SESSION_RESULT_START = "<sessionResult>";

```

Comment:
May be ValidEncounter ?

From: dav5780 [mailto:notifications@github.com]
Sent: Thursday, May 5, 2016 1:31 PM
To: CHIRDL-Openmrs-Modules/chica <chica@noreply.github.com>
Cc: Sarala, Seema <ssarala@iu.edu>; State change <state_change@noreply.github.com>
Subject: Re: [CHIRDL-Openmrs-Modules/chica] Chica 576 force print (#106)


In web/src/org/openmrs/module/chica/web/ManualCheckinSSNMRN.java<https://github.com/CHIRDL-Openmrs-Modules/chica/pull/106#discussion_r62222122>:

> @@ -38,8 +31,6 @@

>       private static final String XML_FALSE = "false";

>       private static final String XML_SESSION_RESULT_START = "<sessionResult>";

Thoughts on the naming of this? session doesn't really seem to fit what we are checking here.

—
You are receiving this because you modified the open/close state.
Reply to this email directly or view it on GitHub<https://github.com/CHIRDL-Openmrs-Modules/chica/pull/106/files/d82c0cc0066f79858f2b6778624fe64d05d0f37f..1c2ed74d9b119261997682c9635e300a0e63b2d9#r62222122>